client/my-sites/earn/memberships/index.jsx

```diff
@@ -302,37 +302,51 @@ class MembershipsSection extends Component {
 		);
 	}
 
-	renderSettings() {
+	renderManagePlans() {
 		return (
 			<div>
-				<SectionHeader label={ this.props.translate( 'Settings' ) } />
-				<CompactCard href={ '/earn/payments-plans/' + this.props.siteSlug }>
+				<SectionHeader label={ this.props.translate( 'Manage plans' ) } />
+				<Card href={ '/earn/payments-plans/' + this.props.siteSlug }>
 					<QueryMembershipProducts siteId={ this.props.siteId } />
-					<div className="memberships__module-products-title">
-						{ this.props.translate( 'Payment plans' ) }
-					</div>
-					<div className="memberships__module-products-list">
-						<Gridicon icon="tag" size={ 12 } className="memberships__module-products-list-icon" />
-						{ this.props.products
```

Comment:
`products` is no longer used, so there is dead code we could remove now:

https://github.com/Automattic/wp-calypso/blob/657f4c125d1bc5ce1953fdec6dcf22509a9427f9/client/my-sites/earn/memberships/index.jsx#L662